draft-ietf-ppm-dap.md

```diff
@@ -1186,27 +1271,157 @@ the last round is reached and each Aggregator computes an output share. All
 told there there are `ceil((ROUNDS+1)/2)` HTTP requests sent, where `ROUNDS` is
 the number of rounds specified by the VDAF.
 
-[OPEN ISSUE: For the moment we don't have an example of a VDAF that requires
-more than two rounds. Eventually we may want to specialize the spec to
-accommodate 1-2 rounds (or even just 1).]
+The state machine for VDAF preparation is shown in {{vdaf-prep-state-machine}}.
+
+~~~ state
+ START --> CONTINUED(round, host_prep_state)
+  |                |                    |
+  |                |                    v
+  +--> REJECTED <--+     FINISHED(out_share)
+~~~
+{: #vdaf-prep-state-machine title="State machine for VDAF preparation."}
+
+State transitions are made when the state is acted upon by the host's local
+inputs and/or a prep step sent by the peer. The initial state is `START`.
+
+~~~ transition
```

Comment:
@branlwyd: Please double check that both of these transition rules align with your understanding.